backend/routes/portfolios.js

```diff
@@ -373,6 +373,36 @@ router.get("/earnings/:id", async (req, res) => {
   );
 });
 
+router.get("/getNotes/:id", async (req, res, next) => {
+  const portfolioId = parseInt(req.params.id);
+  const portfolio = await prisma.portfolio.findUnique({
+    where: {
+      id: portfolioId,
+    },
+  });
+  if (portfolio == null) {
+    next(new BadParams("no portfolio with such Id"));
+  }
+  res.json(portfolio.notesDoc);
+});
+
+router.post("/setNotes/:id", async (req, res, next) => {
+  const portfolioId = parseInt(req.params.id);
+  const newDoc = req.body.html;
+  const portfolio = await prisma.portfolio.update({
```

Comment:
can we make sure the auth user is the owner of the note? in case people overrides the parameter and updating some private portfolios.